plugins/Installation/FormDatabaseSetup.php

```diff
@@ -41,6 +41,9 @@ function init()
         $adapters = array();
         foreach ($availableAdapters as $adapter => $port) {
             $adapters[$adapter] = $adapter;
+            if ($adapter == 'PDO\MYSQL') {
+                $adapters[$adapter] .= ' (' . Piwik::translate('General_Recommended') . ')';
```

Comment:
Instead of adding a test based on the string, maybe you could add a method `isRecommendedAdapter()` to the core/Db/Adapter class, and call this method here?